### PR TITLE
docs(readme): build-ci and test-ci-push badges

### PR DESCRIPTION
**Pull request by 733amir**
Fixes: #631 

## Description of the change:
* Edited existing badge from `ci.yml` to `build-ci.yml`
* Added another badge to `test-ci-push.yml`

## Motivation for the change:
* There was no `ci.yml` anymore.

## How to manually test:
* Open `README.md` and click on the badges.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # cryostat-operator
 
-[![CI build](https://github.com/cryostatio/cryostat-operator/actions/workflows/ci.yaml/badge.svg)](https://github.com/cryostatio/cryostat-operator/actions/workflows/ci.yaml)
+[![CI build](https://github.com/cryostatio/cryostat-operator/actions/workflows/build-ci.yaml/badge.svg)](https://github.com/cryostatio/cryostat-operator/actions/workflows/build-ci.yaml)
```
Comment (tthvo) [inline README.md:3]:
```suggestion
[![CI build](https://github.com/cryostatio/cryostat-operator/actions/workflows/build-ci.yml/badge.svg)](https://github.com/cryostatio/cryostat-operator/actions/workflows/build-ci.yml)
```

This should render as:

[CI build](https://github.com/cryostatio/cryostat-operator/actions/workflows/build-ci.yml)

File: README.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
+[![Test CI push](https://github.com/cryostatio/cryostat-operator/actions/workflows/test-ci-push.yaml/badge.svg)](https://github.com/cryostatio/cryostat-operator/actions/workflows/test-ci-push.yaml)
```
Comment (tthvo) [inline README.md:4]:
```suggestion
[![Test CI push](https://github.com/cryostatio/cryostat-operator/actions/workflows/test-ci-push.yml/badge.svg)](https://github.com/cryostatio/cryostat-operator/actions/workflows/test-ci-push.yml)
```

Rendered as:

[test-CI-on-push](https://github.com/cryostatio/cryostat-operator/actions/workflows/test-ci-push.yml)

**Review by tthvo — COMMENTED** [review]
Thanks for contributing @733amir! Just need to update the file extensions of the workflow files.

Can I also ask you to sign your commits? https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits

**Comment by 733amir** [comment]
> Can I also ask you to sign your commits? https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits

I have signed my commit. GitHub didn't know about my public key which I added to my profile.

**Comment by tthvo** [comment]
Sounds good! Thanks! Its verified now.

File: README.md (hunk 1 above)
Comment (tthvo) [reply README.md:3]: ping @733amir :))

**Review by andrewazores — APPROVED** [review]

**Pull request merged**